hammer/tech/specialcells.py

```diff
@@ -29,9 +29,9 @@ class SpecialCell(BaseModel):
     # Endcap, filler, etc.
     cell_type: CellType
     name: List[str]
-    size: Optional[List[str]]
-    input_ports: Optional[List[str]]
-    output_ports: Optional[List[str]]
+    size: Optional[List[str]] = None
```

Comment:
I think we should avoid changing stuff unless it has functional value, ideally we would be able to upstream parts of this to main Hammer.

Reply:
pydantic will crash the program if this isn't true- this is fixed in upstream hammer I think